cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -43,7 +43,7 @@ func CreateInitStaticPodManifestFiles(manifestDir string, cfg *kubeadmapi.InitCo
 	return CreateStaticPodFiles(manifestDir, cfg, kubeadmconstants.KubeAPIServer, kubeadmconstants.KubeControllerManager, kubeadmconstants.KubeScheduler)
 }
 
-// GetStaticPodSpecs returns all staticPodSpecs actualized to the context of the current InitConfiguration
+// GetStaticPodSpecs returns all staticPodSpecs actualized to the context of the current ClusterConfiguration
 // NB. this methods holds the information about how kubeadm creates static pod manifests.
 func GetStaticPodSpecs(cfg *kubeadmapi.InitConfiguration, k8sVersion *version.Version) map[string]v1.Pod {
```

Comment:
Is there a reason for not switching  `InitConfiguration` into `ClusterConfiguration`\\`APIEndpoint` pair also here? (e.g. similar to what proposed for `getAPIServerCommand`)

Reply:
Must have missed that one. Thanks for pointing it out!